src/controllers/requestController.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable max-len */
+import db from '../database/models';
+import helper from '../utils/helper';
+import sendResult from '../utils/sendResult';
+
+const Request = {
+  async getRequest(req, res) {
+    try {
+      const token = helper.getToken(req);
+      const { userId, verified } = await helper.verifyToken(token);
+      if (!(userId) || verified === undefined) {
+        return sendResult(res, 401, 'Invalid Token');
+      }
+      if (!verified) {
```

Comment:
Create a middleware for checking if a user is verified

Reply:
It is fixed